src/clr-icons/shapes/technology-shapes.ts

```diff
@@ -77,8 +77,8 @@ export const ClrShapeWifi = clrIconSVG(
             <path class="clr-i-solid clr-i-solid-path-3" d="M28.82,15.44a20.59,20.59,0,0,0-21.7,0,1.51,1.51,0,0,0-.46,2.1l.11.17a1.49,1.49,0,0,0,2,.46,17.4,17.4,0,0,1,18.36,0,1.49,1.49,0,0,0,2-.46l.11-.17A1.51,1.51,0,0,0,28.82,15.44Z"/>
             <path class="clr-i-solid clr-i-solid-path-4" d="M24.88,21.49a13.41,13.41,0,0,0-13.82,0,1.5,1.5,0,0,0-.46,2.09l.1.16a1.52,1.52,0,0,0,2.06.44,10.27,10.27,0,0,1,10.42,0,1.52,1.52,0,0,0,2.06-.45l.1-.16A1.49,1.49,0,0,0,24.88,21.49Z"/>
 </svg>`);
-export const ClrShapeRackServer =
-    clrIconSVG(`<rect class="clr-i-outline--alerted clr-i-outline-path-1--alerted" x="10" y="17" width="14" height="2"/>
+export const ClrShapeRackServer = clrIconSVG(
```

Comment:
Is this part of your PR? 
I pulled the changes for your PR locally and `npm run clang:check` failes. `npm run clang:fix` reverts this back to what it used to be.

Reply:
it wasn't originally, but was added when I did `npm run clang:format`.  I was surprised by it, but figured it was probably correct because it was a result from `npm run clang:format`.

Reply:
Hmmm, it sounds like your computer is doing the opposite of mine? 
Anyway, can you revert that and try again?

Reply:
Looks like unit tests are running now. Sorry, I'm not sure exactly whats happening on your system. Thanks for working with us to get the tests to run

Reply:
I was running the clang stuff through **git bash** on windows, don't know if that would result in any differences.  I can try pulling it onto a mac later and see what results from the same scripts there.

Thanks for all the help.